resources/sdk/purecloudgo/templates/api.mustache

```diff
@@ -17,16 +17,15 @@ type {{classname}} struct {
 
 // New{{classname}} creates an API instance using the default configuration
 func New{{classname}}() *{{classname}} {
+	fmt.Sprintf(strings.Title(""), "")
```

Comment:
Putting this redundant function call here to make use of the "fmt" and "strings" packages. They are only used for certain API calls and will cause a compiler error if they are imported and not used